File: Assets/AltUnityTester/AltUnityDriver/Commands/InputActions/MoveMouse.cs
using Assets.AltUnityTester.AltUnityDriver.UnityStruct;

public class MoveMouse : AltBaseCommand
{
    Vector2 location;
    float duration;
    public MoveMouse(SocketSettings socketSettings, Vector2 location, float duration) : base(socketSettings)
    {
        this.location = location;
        this.duration = duration;
    }
    public void Execute(){
        string locationJson = Newtonsoft.Json.JsonConvert.SerializeObject(location, Newtonsoft.Json.Formatting.Indented, new Newtonsoft.Json.JsonSerializerSettings
        {
            ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore
        });
        Socket.Client.Send(toBytes(CreateCommand("moveMouse", locationJson.ToString(), duration.ToString())));
        var data = Recvall();
        if (data.Equals("Ok"))
            return;
        HandleErrors(data);
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/InputActions/move_mouse.py
from altunityrunner.commands.base_command import BaseCommand

class MoveMouse(BaseCommand):
    def __init__(self, socket,request_separator,request_end, x, y, duration):
        super().__init__(socket,request_separator,request_end)
        self.x=x
        self.y=y
        self.duration=duration
    
    def execute(self):
        location = self.vector_to_json_string(self.x, self.y)
        print ('Move mouse to: ' + location)
        data = self.send_data(self.create_command('moveMouse', location, self.duration ))
        return self.handle_errors(data)